### DISCUSSION TITLE
Need code sample for localhost http echo server.

### DISCUSSION THREAD

**Discussion by zydjohnHotmail** [dotnet/WatsonWebserver#56] [General]
Hello:
I need a simple http echo server, which can accept localhost http post data, and show the data posted by http client.
I looked at your repo., and found it rather complicated.
Can you provide a simple http echo server code sample?
By the way, I am using Visual Studio 2019 version 16.9.0 targeting .net 5 on Windows 10.
Thanks,

**Comment by jchristn** [comment]
Hi @zydjohnHotmail just set a route and then:
```
await ctx.Response.Send(ctx.Request.DataAsString());
```
or
```
await ctx.Response.Send(ctx.Request.DataAsBytes());
```

Check out the Test.Default project for a simple example.

Cheers, Joel
